### ISSUE TITLE
Add "readonly" option

### ISSUE THREAD

**Issue by fcfangcc** [abichinger/vue-js-cron#38]
Sometimes we don't allow user to edit it. **readonly** will be necessary.
```html
<cron-light v-model="value" @error="error=$event" readonly></cron-light>
```

**Comment by abichinger** [comment]
Thank you for this suggestion. I'll definitely add that

**Comment by abichinger** [comment]
This feature is available in `@vue-js-cron/light@^3.1.0` 🎉
